rest/rest-smartproxy/src/main/java/org/ow2/proactive_grid_cloud_portal/smartproxy/RestSmartProxyImpl.java

```diff
@@ -462,89 +462,89 @@ public void registerAsListener() throws NotConnectedException, PermissionExcepti
-        return ((ISchedulerClient) _getScheduler()).isJobFinished(jobId);
+        return _getScheduler().isJobFinished(jobId);
     }
 
     @Override
     public JobResult waitForJob(JobId jobId, long timeout)
             throws NotConnectedException, UnknownJobException, PermissionException, TimeoutException {
-        return ((ISchedulerClient) _getScheduler()).waitForJob(jobId, timeout);
+        return _getScheduler().waitForJob(jobId, timeout);
     }
 
     @Override
     public JobResult waitForJob(String jobId, long timeout)
             throws NotConnectedException, UnknownJobException, PermissionException, TimeoutException {
-        return ((ISchedulerClient) _getScheduler()).waitForJob(jobId, timeout);
+        return _getScheduler().waitForJob(jobId, timeout);
     }
 
     @Override
     public boolean isTaskFinished(String jobId, String taskName)
             throws UnknownJobException, NotConnectedException, PermissionException, UnknownTaskException {
-        return ((ISchedulerClient) _getScheduler()).isTaskFinished(jobId, taskName);
+        return _getScheduler().isTaskFinished(jobId, taskName);
     }
 
     @Override
     public TaskResult waitForTask(String jobId, String taskName, long timeout) throws UnknownJobException,
             NotConnectedException, PermissionException, UnknownTaskException, TimeoutException {
-        return ((ISchedulerClient) _getScheduler()).waitForTask(jobId, taskName, timeout);
+        return _getScheduler().waitForTask(jobId, taskName, timeout);
     }
 
     @Override
     public List<JobResult> waitForAllJobs(List<String> jobIds, long timeout)
             throws NotConnectedException, UnknownJobException, PermissionException, TimeoutException {
-        return ((ISchedulerClient) _getScheduler()).waitForAllJobs(jobIds, timeout);
+        return _getScheduler().waitForAllJobs(jobIds, timeout);
     }
 
     @Override
     public Map.Entry<String, JobResult> waitForAnyJob(List<String> jobIds, long timeout)
             throws NotConnectedException, UnknownJobException, PermissionException, TimeoutException {
-        return ((ISchedulerClient) _getScheduler()).waitForAnyJob(jobIds, timeout);
+        return _getScheduler().waitForAnyJob(jobIds, timeout);
     }
 
     @Override
     public Map.Entry<String, TaskResult> waitForAnyTask(String jobId, List<String> taskNames, long timeout)
             throws UnknownJobException, NotConnectedException, PermissionException, UnknownTaskException,
             TimeoutException {
-        return ((ISchedulerClient) _getScheduler()).waitForAnyTask(jobId, taskNames, timeout);
+        return _getScheduler().waitForAnyTask(jobId, taskNames, timeout);
     }
 
     @Override
     public List<Map.Entry<String, TaskResult>> waitForAllTasks(String jobId, List<String> taskNames, long timeout)
             throws UnknownJobException, NotConnectedException, PermissionException, UnknownTaskException,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)